doc/general/operation-modes/operation-modes.md

```diff
@@ -34,6 +34,38 @@ The queue nutshell serves to encapsulate the functionality of a receipt chain fo
 
 The Helper nutshell is used to support updates and configurations.
 
+#### fiskaltrust.SignatureCloud
```

Comment:
SignaturCloud is written without an "e", also applies for below.

Reply:
corrected